README.md

```diff
@@ -121,7 +121,10 @@ Example of possible config.
          }
         ]},
      {ergw_aaa_radius, [
-        {server, {{127,0,0,1}, 1812, <<"secret">>}},
+        {server, #{
```

Comment:
server as key, but possible to have multiple servers in the value. Would servers be a better name? Or is post-fix s only for lists?

Reply:
not sure, maybe it's better to name `backup_servers`?

Reply:
Maybe make sense create new option in the same level as server? Eg:
```erlang
{ergw_aaa_radius, [
    {server, {{127,0,0,1}, 1812, <<"secret">>}},
    {server_backup, {{127,0,0,1}, 1813, <<"secret">>}},
    %% ...
]}
```

Reply:
Yes, server_backup is good.

Reply:
Done.